bin/ansible2gdeploy.py

```diff
@@ -64,33 +96,61 @@ def main():
 
     # get machines from the inventory file
     servers = inventory.options(GLUSTER_SERVER_GROUP)
-    if gdeploy_conf.has_section("clients"):
+    if any([gc.has_section("clients") for gc in gdeploy_confs.values()]):
         clients = inventory.options(GLUSTER_CLIENT_GROUP)
     else:
         clients = []
-    print("servers: " + ", ".join(servers), file=sys.stderr)
-    print("clients: " + ", ".join(clients), file=sys.stderr)
-
-    # ignore hosts section if present
-    if gdeploy_conf.has_section("hosts"):
-        gdeploy_conf.remove_section("hosts")
-    # add servers into hosts sections
-    gdeploy_conf.add_section("hosts")
-    for server in servers:
-        gdeploy_conf.set("hosts", server, None)
 
-    if gdeploy_conf.has_section("clients"):
-        gdeploy_conf.set("clients", "hosts", ",".join(clients))
+    # prepare list of storage devices
+    storage_devices = []
+    if args.storage_devices:
+        storage_devices = args.storage_devices.split(",")
 
-    if args.dry_run:
-        gdeploy_conf.write(sys.stdout)
-    else:
-        if args.output is not None:
-            output_filename = args.output
+    print("servers: " + ", ".join(servers), file=sys.stderr)
+    print("clients: " + ", ".join(clients), file=sys.stderr)
+    print("devices: " + ", ".join(storage_devices), file=sys.stderr)
+
+    # update gdeploy config files
+    for gdeploy_conf_file in gdeploy_confs:
```

Comment:
It would be better to iterate on both key and values of `gdeploy_confs` dictrionary - that simplifies the code in the for loop a bit:

```
for gdeploy_filename, gdeploy_conf in gdeploy_confs.items():
   ...
```

Then you can use `gdeploy_conf` instead of  `gdeploy_confs[gdeploy_conf_file]`

Reply:
Good point, fixed by https://github.com/usmqe/usmqe-setup/pull/172/commits/064447e645de405bdeaff333d756b6caa9ea53c0